core/src/main/scala/kafka/log/LogManager.scala

```diff
@@ -308,11 +312,11 @@ class LogManager(logDirs: Seq[File],
         threadPools.append(pool)
 
         val cleanShutdownFile = new File(dir, Log.CleanShutdownFile)
-
         if (cleanShutdownFile.exists) {
-          debug(s"Found clean shutdown file. Skipping recovery for all logs in data directory: ${dir.getAbsolutePath}")
+          info(s"Skipping recovery for all logs in $dir since clean shutdown file was found")
```

Comment:
Have we checked that the `dir.toString` does what we want? If the path is relative, I think it will only print that.